### PR TITLE
If there is a network issue, manual mode will still be present. Feature #3855

### PR DESCRIPTION
**Pull request by jacobvanorder**
Regardless of if there's a network error or even if there are endpoints, the manual endpoint is always created. The timeout setting for the network request is now 20 seconds.

### CODE REVIEW

File: NCSNavField/RootViewController.h [hunk 1]
```diff
@@ -55,6 +55,7 @@
 - (void)deleteButtonWasPressed;
 - (void)unloadSurveyor:(Instrument*)instrument;
 - (void)syncContacts:(CasServiceTicket*)serviceTicket withRetrieval:(BOOL)shouldRetrieve;
+- (IBAction)screenParticipant:(UIButton *)button;
```
Comment (jdzak) [inline NCSNavField/RootViewController.h]: This doesn't seem related.

Comment (jacobvanorder) [inline NCSNavField/RootViewController.h]: It's not, it's a carry over from the other branch. Will fix and repush.
